dax/api.go

```diff
@@ -16,11 +16,12 @@
 package dax
 
 import (
+	"io"
```

Comment:
I'll revert this as this change has nothing to do with the fix - it was something that my formatter did.